src/modules/ProductsPage/ProductsList/ProductsList.tsx

```diff
@@ -8,7 +8,7 @@ import { ProductsCategory } from '@/types/ProductsCategory';
 import { ProductCard } from '@/modules/shared/components/ProductCard';
 
 type Props = {
-  products: { [id: string]: Product };
+  products: Product[];
```

Comment:
if you change  product type, you need change `{Object.values(products).map(product` to {products.map(product